oceanspy/_oceandataset.py

```diff
@@ -830,9 +830,10 @@ def create_tree(self, grid_pos="C"):
             z = _xr.zeros_like(Y)
 
         # Stack
-        x_stack = x.stack(points=x.dims).values
-        y_stack = y.stack(points=y.dims).values
-        z_stack = z.stack(points=z.dims).values
+        rid_value = 777777
+        x_stack = x.stack(points=x.dims).fillna(rid_value).values
+        y_stack = y.stack(points=y.dims).fillna(rid_value).values
+        z_stack = z.stack(points=z.dims).fillna(rid_value).values
```

Comment:
Does it work if you use `.data` instead of `.values`? That way dask arrays are not loaded into memory.

Reply:
I haven't tested that. But since the very next step is to feed the whole thing into scipy, I think laziness is not really necessary here. I believe it is also values in the original code

Reply:
Let's use `.data` if it works. It wasn't available when we first started working on oceanspy and it is possible that SciPy is/will be able to use dask laziness